iphone/Classes/TiContactsPerson.m

```diff
@@ -958,6 +967,11 @@ -(CNSaveRequest*)getSaveRequestForRemoveFromGroup: (CNMutableGroup*) group
 	return saveRequest;
 }
 
-@end
+- (void)checkAndNotifyObserver {
+	if (self.observer && [self.observer respondsToSelector:@selector(didUpdatePerson:)]) {
```

Comment:
For the next time: Use the Obj-C getter `[self observer]` here instead of the point-notation. Pedro once told me it's better code-style :blush: